docs/decisions/0002-include-test-cases-files.rst

```diff
@@ -0,0 +1,89 @@
+###################
```

Comment:
I don't think the decision folder is the place for that documentation. 
https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0019-bp-developer-documentation.html#adrs
Maybe is a good idea to create a folder like help_for_devs or dev_tools or something like that. Later the information will be structured but definitely what is being added is not a decision.

What do you think?

Reply:
That seems fine to me. I have created a `help_for_devs` folder where the instructions for using the json files are.

Thanks @MaferMazu for your note!!!